python/run.py

```diff
@@ -302,14 +323,26 @@
             elif args.measure[0]=='tran_3':
                 CCEs1[i], CCEs2[i], CCEs3[i] = draw3MultiCCE(axs[1],data1.X,data1.Y,data1.Y1,data1.Y2,final_let,lines[0])
                 axs[1].set_ylabel(r'CCE $\times$ '+str(args.scaleLET))
-                #axs[0].set_xlim(-1,10) #lorenzo
+                axs[0].set_xlim(-1,10) #lorenzo
```

Comment:
Ciao Lorenzo, thanks for the additions!
Could you please remove this line of a fixed time axis range? I've added the possibility to use instead the --maxX flag to set a smaller time range.